Sage/carbon
# chore: remove cypress split and cypress mochawesome reporter

### Proposed behaviour

- Remove `cypress-split` so remaining cypress tests are run serially in CI
- Remove `cypress-mochawesome-reporter` and replace with list reporter

### Current behaviour

- Due to having fewer cypress tests, our Cypress CI checks are failing, as one of the parallelised CI jobs cannot find any tests. [Link to example PR ](https://github.com/Sage/carbon/actions/runs/7798231991?pr=6568)

### Checklist




#### QA


### Additional context



### Testing instructions

- Check Cypress tests are only run serially in CI
- Check CI output of test results is clear and understandable
